extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -87,7 +89,7 @@
     private JsonObject playerArgs;
     @Nonnull
     private final Map<String, String> videoInfoPage = new HashMap<>();
-    private JsonObject playerResponse;
+    public JsonObject playerResponse;
```

Comment:
why public?

Reply:
My bad, I used it for tests, forgot to let it private again